arcadia-parties.sln

```diff
@@ -0,0 +1,34 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio 15
+VisualStudioVersion = 15.0.26124.0
```

Comment:
There is a strange version here. All my projects created in MSVS2019 use VisualStudioVersion = 16.X here...

Reply:
I've fixed this.
Removed lines